webhook.go

```diff
@@ -429,9 +519,21 @@ func (whsvr *WebhookServer) mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admissi
 	glog.Infof("AdmissionReview for Kind=%v, Namespace=%v Name=%v (%v) UID=%v patchOperation=%v UserInfo=%v",
 		req.Kind, req.Namespace, req.Name, pod.Name, req.UID, req.Operation, req.UserInfo)
 
+	safe, err := isSafe(&pod, string(req.Operation))
```

Comment:
err needs to be checked before safe is processed. EDIT: Refer to above comment on use of error.